### PR TITLE
fade effect for subcategory bar

### PR DESCRIPTION
**Pull request by danielfmiranda**
# Description
Related PRs/issues: #9871 
Link to sample test page: http://localhost:8000/en/privacynotincluded/categories/smart-home/

This PR adds the fade effect on the end of the subcategories for medium sized screens and above. 
Figma designs can be found here: [Figma](https://www.figma.com/file/GwdoUiY4fzVyeLfXsAJGPq/*Privacy-Not-Included-Archive-2022?node-id=5609%253A102175)

# Screenshots

## L - XL:

**Beginning of list:**
Screenshot 2023-02-01 at 17-41-29 Privacy Not Included A Buyer’s Guide for Connected Products

**End of list:**
Screenshot 2023-02-01 at 17-41-53 Privacy Not Included A Buyer’s Guide for Connected Products

## M

**Beginning of list:**
206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a

**End of list:**
206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a

## S

**Beginning of list:**
206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a

**End of list:**
206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a

## XS

**Beginning of list:**

206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a

**End of list:**

206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a

# Steps to test

1. Check this branch out
2. run `inv copy-prod-db
3. Visit http://localhost:8000/en/privacynotincluded/categories/smart-home/
4. If the subcategory nav below is looking as expected on all screen sizes, testing is complete!

### CODE REVIEW

**Review by nancyt1 — APPROVED** [review]
Love it! Thank you!

**Review by fessehaye — APPROVED** [review]
The three spacing units will be affected by the tailwind spacing PR FYI. You might have to add a fix to this after it gets merged. Looks great though!

**Comment by danielfmiranda** [comment]
Thanks for the heads up @fessehaye!

I have updated the spacing classes so the spacing can remain at 12px, and will not merge this ticket until the new spacing ticket #9943 is merged in.

**Pull request merged**
